src/sentry/models/integration.py

```diff
@@ -63,6 +63,9 @@ def get_provider(self):
         from sentry import integrations
         return integrations.get(self.provider)
 
+    def get_integration(self):
```

Comment:
i feel like this could have a slightly better name since `Integration.get_integration` is weird. open to suggestions

Reply:
I think we came to the conclusion to go for `get_installation` based on discussion in Slack.